app/Http/Controllers/Api/Web/StatsController.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+/**
+ * Pony.fm - A community for pony fan music.
+ * Copyright (C) 2015 Peter Deltchev
```

Comment:
Unless assigning the copyright to me was your intent, put your own name and the current year in license headers for files you create or make significant changes to. If another name is already in such a notice, you can add another copyright line for yourself.